client/game/TaskStimulus.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+export default class TaskStimulus extends React.Component {
+  render() {
+    const { round, stage, player } = this.props;
+
+    window.round = round;
+
+    const imagePath = round.get("imagePath");
+    const questionText = round.get("questionText");
+
+    return (
+      <div className="task-stimulus">
+        <div className="task-image">
+          {imagePath == undefined ? (
```

Comment:
```suggestion
          {imagePath ? (
```

If `imagePath` is undefined, it is also false. Not comparing to undefined will also have the added advantage of catching other unfortunate cases of bad values such as "" (empty string) or null.

Reply:
TIL